src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -940,6 +949,10 @@ run_test_suite_device(const std::shared_ptr<xrt_core::device>& device,
 
     if(schemaVersion == Report::SchemaVersion::text)
       pretty_print_test_run(ptTest, status, _ostream);
+    
+    //flash shell support: if verify xclbin is not present, exit immediately
+    if(ptTest.get<std::string>("name").compare("Verify kernel") == 0 && ptTest.get<std::string>("status").compare("skipped") == 0)
```

Comment:
Again, what is so special about the "test name".  Inquiry minds what to know :^).